manifest-celery.yml.j2

```diff
@@ -23,6 +23,7 @@ applications:
     AWS_ACCESS_KEY_ID: {{ AWS_ACCESS_KEY_ID }}
     AWS_SECRET_ACCESS_KEY: {{ AWS_SECRET_ACCESS_KEY }}
 
+    TEMPLATE_PREVIEW_INTERNAL_SECRETS: {{ TEMPLATE_PREVIEW_INTERNAL_SECRETS | tojson }}
     TEMPLATE_PREVIEW_API_KEY: {{ TEMPLATE_PREVIEW_API_KEY }}
```

Comment:
Does TEMPLATE_PREVIEW_API_KEY need to be in the manifests still?

Reply:
Nope, I've removed it being used and fixed the makefile to take `TEMPLATE_PREVIEW_INTERNAL_SECRETS` too.

Note, whilst template preview will no longer use `TEMPLATE_PREVIEW_API_KEY`, our other apps e.g. the api will still take this secret as it will be the single api key it uses to talk to template preview